smriprep/workflows/anatomical.py

```diff
@@ -47,7 +47,7 @@ def init_anat_preproc_wf(
       - Brain extraction and INU (bias field) correction.
       - Brain tissue segmentation.
       - Spatial normalization to standard spaces.
-      - Surface reconstruction with FreeSurfer_.
+      - Surface reconstruction with FreeSurfer.
```

Comment:
I think you need to include `links.rst` to be able to use the reference.

Reply:
I haven't been able to figure out where the include should be called.

Reply:
Added a commit.